### DISCUSSION TITLE
Unable to create a page as '/ab-cd/topic1'

### DISCUSSION THREAD

**Discussion by patrickdung** [requarks/wiki#4769] [Error / Bug Report] [answered]
I don't understand why for some URL, the select button is dimmed.
![image](https://user-images.githubusercontent.com/38665827/145282895-62155ad8-df6f-406e-8462-03350e2a0c19.png)

Not ok:
/ab-cd/topic1

OK to create:
/ab-c/topic1
/ab-cde/topic1

Any ideas?

**Comment by NGPixel** [comment, marked as answer]
XX and XX-XX are reserved paths for locales, as detailed in the [docs](https://docs.requarks.io/guide/pages#naming-restrictions).

**Reply by patrickdung** [reply]
Thanks. I was going to create a path like '/ci-cd/some-topics', which is something about CICD/pipeline.
